php/classes/integrations/elementor/widgets/elementor-subscribe-buttons-widget.php

```diff
@@ -2,9 +2,36 @@
 
 namespace SeriouslySimplePodcasting\Integrations\Elementor\Widgets;
 
+use Exception;
 use SeriouslySimplePodcasting\Controllers\Players_Controller;
+use SeriouslySimplePodcasting\Helpers\Log_Helper;
```

Comment:
I don't see where this is used in the code below, am I missing something? Loading in classes that don't get used isn't a big deal but I prefer to avoid it if I remember to check

Reply:
Good call, thanks